ports/servo/main.rs

```diff
@@ -142,7 +142,7 @@ fn main() {
     // Our wrapper around `Browser` that also implements some
     // callbacks required by the glutin window implementation.
     let mut browser = BrowserWrapper {
-        browser: Browser::new(window.clone()),
+        browser: Browser::new(window.clone(), opts::get().url.clone().unwrap()),
```

Comment:
`.unwrap_or(ServoUrl::parse("about:blank").unwrap())` may make sense here. If that unwrap fails, then we're in deeply screwed territory anyway so it doesn't matter.